src/tutorials/lua_basics_lightbulb.md

```diff
@@ -365,398 +390,227 @@ switchTrigger.interactedEvent:Connect(OnSwitchInteraction)
-
-### Find the Correct Location for the Light
-
-The next step is to change the position of the light when it spawns.
-
-1. Open the **Light bulb** group in the Hierarchy and find the group named **Bulb**.
-2. When you select **Bulb**, the gizmo should be positioned in the center of the lightbulb, so you can use it to determine the light's position.
-
-!!! note
-    We could look up the global position of the **Bulb** and save it, the same way we found the rotation for the switch, but that would mean if you ever moved the lightbulb the light would not move with it. You would need to update the code yourself any time your move things. Instead we can find the bulb's position in the script and tell it to always spawn there, wherever that is.
-
-Add these variables to the top of your script:
-
-```lua
-local bulb = World.FindObjectByName("Bulb")
-local bulbPosition = bulb:GetWorldPosition()
-```
-
-- `bulb` is the name of our variable referencing the object
-- `FindObjectByName` is a Core function to find objects you wish to reference.
-- `bulbPosition` is the name of our variable defining where we want the light placed.
-- `bulb:GetWorldPosition()` gets the coordinates of the bulb object.
-
-### Improve the Light Location Code
-
-The `GetWorldPosition` function is very powerful, but will break if you name anything else in your entire project **Bulb**. This is not a problem in an empty project, but could be if, for example, you built a whole house full of lights and bulbs.
-
-Instead, you can look for the specific **Bulb** group that is part of this template, which will be the one that is a **child** of the template.
-
-```lua
-local lightBulbFolder = script:FindAncestorByName("Lightbulb & Switch")
-local bulb = lightBulbFolder:FindDescendantByName("Bulb")
-local bulbPosition = bulb:GetWorldPosition()
-```
-
-`World.FindObjectByName` still does the same thing, however.
+As your scripts get longer, these practices will make them easier to read and edit.
 
-### Spawn the Light in the Correct Position
+## Turning the Light On
 
-The next step is to update the `SpawnAsset` function to spawn the light wherever the **Bulb** is. Fortunately, `World.SpawnAsset`, takes extra **optional parameters**, which can be added in `{}`.
+Your next project goal is to make the switch more functional and have it turn on a light when players interact with it.
 
-Find the `SpawnAsset` function and change it to include the bulbPosition as the position:
+In the `OnTriggerInteracted()` function under our `RotateTo()` statement, type:
 
 ```lua
-     World.SpawnAsset(propLightTemplate, { position = bulbPosition })
+Light.visibility = Visibility.INHERIT
 ```
 
-!!! hint "Be sure to use a comma `,` between the inputs into the function!"
+- `Light` is the object we are trying to alter.
+- `visiblity` is the property we are trying to change.
+- `Visibility` is an enum that contains states of visibility of an object: `FORCE_ON`, `FORCE_OFF`, or `INHERIT`.
```

Comment:
Don't say enum, explain what it is instead. 

"Set of possible values". We don't even really need to get across that it's a number.